src/Microsoft.AspNetCore.ApplicationInsights.HostingStartup/JavaScriptSnippetTagHelperComponent.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.ApplicationInsights.AspNetCore;
+using Microsoft.AspNetCore.Razor.TagHelpers;
+
+namespace Microsoft.AspNetCore.ApplicationInsights.HostingStartup
+{
+    /// <summary>
+    /// The <see cref="TagHelperComponent"/> that injects the <see cref="JavaScriptSnippet"/> to the head tag.
```

Comment:
`at the end of the`. Also might need to change head => body depending on a response to my other comment.